### ISSUE TITLE
Password Reset Functionality

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/bestevidence#246]
As a user
I want to be able to reset my password if I forget it
By pressing a link on the login page that sends me an email to reset my password, allowing me to log in again

**Comment by ajburls** [comment]
@iteles @Danwhy This version comes up with a screen to ask for a password reset but doesn't send the email. I have waited several hours, checked email is correct and looked in junk folder, but no recovery email received.

**Comment by iteles** [comment]
@ajburls You're right, now that I'm reviewing everything in detail it would seem that the work for this was not completed - just initiated with a small amount of time we had left over from our Sprint 5 budget. I will investigate this this week.
